docs/user_guide/HOWTO/Using_the_SIMP_Platform_RPM.rst

```diff
@@ -0,0 +1,133 @@
+                           the next Major release.
+
+Major.Minor                Downloads the latest Major.Minor          '6.4' will install the latest version of SIMP 6.4,
+                           release of SIMP, updating to all Patch    6.4, upgrading only the Patch releases, not
+                           and Iteration releases in the Minor       upgrading to SIMP 6.5.
+                           release.
+
+Major.Minor.Patch          Downloads the latest Major.Minor.Patch    '6.4.0' will install SIMP 6.4.0, and update to
+                           release of SIMP, only updating if an      any iteration updates, but not update to
+                           iteration release is available.           SIMP 6.4.1.
+
+Major.Minor.Path-Iteration Downloads the specific version of SIMP,   '6.4.0-0' will install SIMP 6.4.0-0 and never
+                           never updating to a newer version.        update.
+========================== ========================================= ==================================================
+
+SIMP Release Types
+******************
+
+The ``/etc/yum/vars/simpreleasetype`` variable file controls the type of
+releases you want to download.
+
+=================================== ===========================================
+Option (exact contents of the file) Description
+=================================== ===========================================
+releases                            This setting will have the repos grab only
+                                    fully tested SIMP releases.
+
+rolling                             This setting will have the repos grab
+                                    updates to RPMs that have not yet made it
+                                    into a SIMP Release, but have been tested
+                                    and released individually with confidence.
+                                    This is an example where you can specify a
+                                    specific SIMP release (ex: '6.4.0-0'), but
+                                    still receive updates.
+
+unstable/simp6                      This setting will have the repos grab all
+                                    updates to RPMs in the unstable repo
+                                    (This is extremely dangerous and not
+                                    recommended for production environments).
+                                    'unstable/simp7' will become available
+                                    in the future when SIMP 7 is released.
+=================================== ===========================================
+
+These variables allow you to control the exact updates you receive for SIMP,
+and provide a dynamic system that won't need to be updated or re-installed for
+future versions of SIMP.
+
+.. warning::
+
+  Setting these Yum Var files to invalid contents will break the repo files and
+  prevent successful downloads. The simprelease file should only include numbers,
```

Comment:
Probably should backtick `simprelease`

Reply:
Indentation on this needs to be fixed. The first character of the block must align with the `w` in `warning`

Reply:
Also, please upcase `warning`  to match the rest of the docs.